eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -3,20 +3,33 @@ parameters:
   EnvVars: {}
   MaxParallel: 0
   Matrix:
-    Win2016:
+    Linux - Java 8:
+      OSName: 'Linux'
+      OSVmImage: 'ubuntu-16.04'
+      JavaVersion: '1.8'
+    macOS - Java 8:
+      OSName: 'macOS'
+      OSVmImage: 'macOS-10.13'
+      JavaVersion: '1.8'
+    Windows - Java 8:
+      OSName: 'Windows'
       OSVmImage: 'windows-2019'
-      DisplayName: 'Run Live tests'
+      JavaVersion: '1.8'
+    Linux - Java 11:
+      OSName: 'Linux'
+      OSVmImage: 'ubuntu-16.04'
+      JavaVersion: '1.11'
+    macOS - Java 11:
+      OSName: 'macOS'
+      OSVmImage: 'macOS-10.13'
+      JavaVersion: '1.11'
+    Windows - Java 11:
+      OSName: 'Windows'
+      OSVmImage: 'windows-2019'
+      JavaVersion: '1.11'
   PreRunSteps: []
   TestName: LiveTest
   TimeoutInMinutes: 60
-  TestStepMavenInputs:
```

Comment:
`sdk/cosmos/tests.yml` uses `TestStepMavenInputs` right now because its goal is `verify` instead of `test`. Seems like this could be standardized to `test` or something like it. 

The cosmos live tests also use options specified in the matrix https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/tests.yml#L54 ... these are used to configure the test scenarios. However, I wonder if we can append the `DefaultOptions` from https://github.com/Azure/azure-sdk-for-java/blob/master/eng/pipelines/templates/variables/globals.yml#L2 (though this is probably less important) 

Until the cosmos tests can be made more uniform with our specs we should keep `TestStepMavenInputs` as a parameter. 

@kushagraThapar -- what's needed to switch the goal from `verify` to `test` and would there be any drawbacks to appending our `DefaultOptions` to the maven options?